deployTest/pom.xml

```diff
@@ -0,0 +1,57 @@
+<project xmlns="http://maven.apache.org/POM/4.0.0"
+	xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+	xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/maven-v4_0_0.xsd">
+	<modelVersion>4.0.0</modelVersion>
+
+	<parent>
+		<groupId>org.ibissource</groupId>
+		<artifactId>ibis-adapterframework-parent</artifactId>
+		<version>7.6-SNAPSHOT</version>
+	</parent>
+
+	<artifactId>ibis-adapterframework-assemble-test</artifactId>
+	<name>Ibis AdapterFramework assemble test</name>
+
+	<properties>
+		<mavenDependencyPluginVersion>3.1.2</mavenDependencyPluginVersion>
+	</properties>
+	<dependencies>
+		<dependency>
+			<groupId>org.ibissource</groupId>
+			<artifactId>ibis-adapterframework-test</artifactId>
+			<version>7.6-SNAPSHOT</version>
+			<type>war</type>
+		</dependency>
+		<dependency>
+			<groupId>org.apache.geronimo.specs</groupId>
+			<artifactId>geronimo-jms_1.1_spec</artifactId>
+		</dependency>
+		<dependency>
+			<groupId>commons-dbcp</groupId>
+			<artifactId>commons-dbcp</artifactId>
+		</dependency>
+		<dependency>
+			<groupId>commons-pool</groupId>
+			<artifactId>commons-pool</artifactId>
+		</dependency>
+	</dependencies>
+	<build>
+		<plugins>
+			<plugin>
+				<groupId>org.apache.maven.plugins</groupId>
+				<artifactId>maven-dependency-plugin</artifactId>
+				<version>${mavenDependencyPluginVersion}</version>
+				<executions>
+					<execution>
+						<goals>
+							<goal>copy-dependencies</goal>
+						</goals>
+						<configuration>
+							<excludeTransitive>true</excludeTransitive>
+						</configuration>
+					</execution>
+				</executions>
+			</plugin>
+		</plugins>
+	</build>
```

Comment:
Kan je niet beter pom.type = war gebruiken?

Reply:
Bedoel je in plaats van dat ik transitieve dependencies weglaat? Ik denk het niet, want ik wil met dit project juist niet alleen de .war ophalen. Het gaat ook om de .jar files die ik hier als dependencies opgeef. Maar ik wil niet alle dependencies van het FF! neerzetten, want die zitten al in de .war.

Reply:
Ik snap het. Ik had het project deployTest zo gemaakt dat hij de .war en losse .jar files naast elkaar zet in een subdirectory van target. Maar ik begrijp nu dat we gewoon de .war van ibis4test groter moeten maken. Daar heb ik net een commit voor gedaan.